judge/judgedaemon.main.php

```diff
@@ -253,16 +251,19 @@ function read_judgehostlog(int $n = 20) : string
 function fetch_executable(
     string $workdirpath, string $type, string $execid, bool $combined_run_compare = false) : array
 {
+    // Safe when you trust this code) (workdirpath+type+execid is safe)
```

Comment:
remove first )

Reply:
The comments were only for me, I was planning to remove them when merging.